tool/state-processor/src/tests.rs

```diff
@@ -125,14 +124,15 @@ fn solo_chain_substrate_account_adjust() {
 		let solo_account = tester.solo_accounts.get(test_addr).unwrap();
```

Comment:
I think we could add more samples(`test_addr`) here.

Reply:
Any suggestions about the other typical `test_addr`? There are some test cases below cover the evm accounts.